### ISSUE TITLE
Detect is detecting Opera as Chrome

### ISSUE THREAD

**Issue by iranreyes** [Jam3/generator-jam3#280]
![image](https://cloud.githubusercontent.com/assets/5953115/25030921/191d74d4-209f-11e7-9e13-106559ae1335.png)

We need to check for the string OPR

**Comment by namniak** [comment]
but it's pretty much chrome. are there any issues with it specifically? any unsupported features, or CSS? I was sure Opera is Chrome a couple of versions back

**Comment by jeremenichelli** [comment]
Well, Opera is using Chrome's Blink as rendering engine but the idea of the `detect` library is to specify the browser vendor so it should return Opera in case you need to a specific fix for it, for whatever reason that could be.

**Comment by namniak** [comment]
That's true. I agree that `detect` needs to be able to do that. Also I noticed Edge appears as `undefined` recently. 

I was just wondering about something else - problems or features in Opera specifically that can mess up sites. If we use `detect` to  show unsupported page in Opera, say using `node.js` or it's equivalent in php, then probably we should not because there are no differences between Chrome and Opera. Checked 5-6 sites I've worked in the past year and a half - no issues found. 💯
 
But this thing led to another problem - Chromium. There are def more differences between Chrome and Chromium (which doesn't support mp3 & mp4 and both have exact same UA string). All of those sites ^^ are fucked in Chromium because of the codecs and we usually have media and the experience is build around it. Black screens, errors or can't pass the preloader because of the Promises of media loading. 🙃
